docoptTests/DocoptTestCaseParser.swift

```diff
@@ -24,10 +24,15 @@ public struct DocoptTestCaseParser {
     }
     
     private func removeComments(string: String) -> String {
-        let removeCommentsRegEx: NSRegularExpression? = NSRegularExpression(pattern: "(?m)#.*$", options: .allZeros, error: nil)
+        let removeCommentsRegEx: NSRegularExpression?
```

Comment:
Is it possible to use `guard` here? Something like

```
guard let removeCommentsRegEx = try NSRegularExpression(pattern: "(?m)#.*$", options: []) else {
    return string;
}

let fullRange: NSRange = NSMakeRange(0, string.characters.count)
return removeCommentsRegEx.stringByReplacingMatchesInString(string, options: [], range: fullRange, withTemplate: "")
```

Unfortunately I don't have recent xcode beta installed, so I can't check myself.